tests/src/e2e/ledger_tests.rs

```diff
@@ -4663,6 +4664,52 @@ fn implicit_account_reveal_pk() -> Result<()> {
     Ok(())
 }
 
+#[test]
+fn test_epoch_sleep() -> Result<()> {
```

Comment:
the new test has to be added to `.github/workflows/scripts/e2e.json` to run in CI